packages/sync-actions/src/types-actions.js

```diff
@@ -136,7 +136,7 @@ export function actionsMapFieldDefinitions(
           label: extractedPairs.newObj.label,
           fieldName: extractedPairs.oldObj.name,
         })
-      } else if (diffValue.type.values) {
+      } else if (diffValue?.type?.values) {
```

Comment:
That one fixes this issue https://github.com/commercetools/nodejs/issues/1766

Reply:
good catch! although this is not throwing error anymore, i think the `required` field is still not being updated.
so [this](https://github.com/commercetools/commercetools-importer/pull/2552/files#diff-b9adf055e61d9e3c62b4ffb433f171742bd5f8b04d226bc3919c0c257d476ac5R75-R97) particular test is not failing anymore, but if we check the `required` field, its still `false` if im not mistaken.

Reply:
Yep very true, this field is unchangeable, so the current behaviour is just ignore these fields

Reply:
oh i see! good then 👍